### ISSUE TITLE
Some Material Symbols icons are missing from https://fonts.google.com/icons

### ISSUE THREAD

**Issue by ddereszkiewicz** [google/material-design-icons#1735]
Some Material Symbols icons e.g., `note`, `expand_more` are present in the font set, but are missing from https://fonts.google.com/icons.
Why is this the case? Are those icons safe to use, or are they deprecated and need to be changed?

**Comment by tphinney** [comment]
tl;dr: safe.

Once Material Symbols supports an icon, absent something very unusual (such as a legal reason), the plan so far has been that we support it essentially forever.

We really try to avoid breaking stuff for users, although sometimes fixing a bug in the glyph for an icon will effectively do so.  :(

Now, if a better name comes along, or maybe an existing icon gets tweaked to be identical to another one, sometimes the front end folks decide (either right away or later) to only display those icons under one name in the front end user interface, suppressing the “old” one.

But even then, in the font we either (1) just keep the duplicate icon, or (2) delete it, **but** route its codepoint and ligature to point at another icon, so it is **as if** it was a separate icon, even though under the hood (like, if you crack open the font and browse its glyphs) there is only one base glyph.
